### PR TITLE
Update email.md

### PR DESCRIPTION
**Pull request by ikeryo1182**
Hi.

fix typography in email.md

## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [x] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [x] Documentation has been added (Run `npm run docs`)
- [x] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: docs/helpers/Puppeteer.md [hunk 1]
```diff
@@ -1879,11 +1879,11 @@ I.waitForRequest(request => request.url() === 'http://example.com' && request.me
 
 ### waitForResponse
 
-Waits for a network request.
+Waits for a network response.
```
Comment (ikeryo1182) [inline docs/helpers/Puppeteer.md:1882]: Related #2632

File: docs/plugins.md [hunk 2]
```diff
@@ -388,7 +388,7 @@ const When = () => step`When`;
 const Then = () => step`Then`;
 ```
 
-Scenario('project update test', async ({ I }) => {
+Scenario('project update test', async (I) => {
```
Comment (ikeryo1182) [inline docs/plugins.md:391]: Related #2632

File: docs/helpers/Puppeteer.md (hunk 1 above)
Comment (PeterNgTr) [reply docs/helpers/Puppeteer.md:1882]: oh, if that PR is merged, why do we have them here again, thou?

Comment (ikeryo1182) [reply docs/helpers/Puppeteer.md:1882]: Hmm...
I ran `npm run docs` command, and then this changes occured.
So, I suspend that PR ( #2632 ) was merged but the command, i.e. `npm run docs` , was not ran.

If this changes is not unnecessary,   should I delete this changes ?

**Pull request merged**